docs/platform/visual/analyzer-future-viz-api.md

```diff
@@ -176,7 +176,7 @@ the possible _maximum number of results_ for "Bar chart" visualizations.
 
 | Current Vis. Id.      | Future Vis. Id.                               | Description          |
```

Comment:
Just marking this place to update when we decide what nomenclature to use _current_/_future etc.